src/Schema/Exception/TooManyValidSchemas.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace League\OpenAPIValidation\Schema\Exception;
+
+use cebe\openapi\spec\Schema;
+use Throwable;
+
+// Indicates that data was not matched against a combination of schema keywords
```

Comment:
I think comment is outdated. Thats the problem of all these comments. maybe drop them?

Reply:
you're right; removed it.